snuba/settings/settings_test_distributed_migrations.py

```diff
@@ -17,7 +17,7 @@
         "distributed_cluster_name": "query_cluster",
     },
     {
-        "host": "clickhouse-query",
+        "host": os.environ.get("CLICKHOUSE_MIGRATIONS_HOST", "clickhouse-query"),
```

Comment:
so since we set `CLICKHOUSE_HOST` here https://github.com/getsentry/snuba/blob/174398093727ebf337fc3a781aa727638eef8cde/docker-compose.gcb.yml#L27

this wont' default to `clickhouse-query`. so Im changing the env var to `CLICKHOUSE_MIGRATIONS_HOST` so that by default it's not set, but for the clickhouse-setup testing to work you can specify the host